ButtonStateless/index.jsx

```diff
@@ -51,6 +51,7 @@ const Button = ({
       fontFamily,
       fontSize: fontSizeSmall,
       fontWeight,
+      lineHeight: '0',
```

Comment:
Thanks for the heads up here! I'll make this change with the other work I'm doing in this repo today 😁